### PR TITLE
GH-26 adding `osgi`: an osgi bundle to ease the use of these sdk jars within AEM

### PR DESCRIPTION
**Pull request by francoisledroff**
adding an osgi bundle to ease the use of these sdk jars within AEM

### CODE REVIEW

File: osgi/pom.xml [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+      </plugin>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>3.5.0</version>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <_noee>true</_noee>
+            <!-- <Bundle-Activator>com.adobe.aio.lib.Activator</Bundle-Activator> -->
```
Comment (shikhartanwar) [inline osgi/pom.xml:137]: Should this be uncommented?

File: pom.xml [hunk 2]
```diff
@@ -27,6 +27,7 @@
     <module>events_mgmt</module>
     <module>events_ingress</module>
     <module>events_journal</module>
+    <module>osgi</module>
```
Comment (shikhartanwar) [inline pom.xml:30]: Do we want to add this module to the `README.md` as well?

**Review by shikhartanwar — COMMENTED** [review]
Added minor comments.

File: osgi/pom.xml (hunk 1 above)
Comment (francoisledroff) [reply osgi/pom.xml]: yes

File: pom.xml (hunk 2 above)
Comment (francoisledroff) [reply pom.xml:30]: yes

**Pull request merged**